lib/discovery.js

```diff
@@ -456,5 +389,15 @@ function mixinDiscovery(MsSQL) {
       default:
         return 'String';
     }
-  }
+  };
+
+  MsSQL.prototype.setDefaultOptions = function(options) {
```

Comment:
Do we need to have the empty impls? They seem to be good candidates to have default impl in SQLConnector.